src/scripts/main.js

```diff
@@ -355,6 +355,40 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people); // you can remove it
 
 // write your code here
+function main() {
+  const table = document.documentElement.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the DOM before attempting to append rows to it. If the element is not found, `querySelector` will return `null`, which will cause an error when trying to append rows. Consider adding a check to handle this scenario.